geomesa-hbase/geomesa-hbase-spark-runtime-hbase1/pom.xml

```diff
@@ -59,6 +59,12 @@
             <version>${hbase.version}</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
```

Comment:
Are we including it in order to relocate Guava?

Reply:
right